presentation/src/main/java/com/playone/mobile/presentation/login/LoginPlayonePresenter.kt

```diff
@@ -1,8 +1,18 @@
 package com.playone.mobile.presentation.login
 
+import com.playone.mobile.domain.Credential
 import com.playone.mobile.domain.interactor.auth.SignUpAndSignIn
+import com.playone.mobile.domain.model.User
+import com.playone.mobile.presentation.ViewResponse
+import com.playone.mobile.presentation.mapper.UserMapper
+import io.reactivex.observers.DisposableSingleObserver
 
-class LoginPlayonePresenter(val signUpAndSignIn: SignUpAndSignIn) : LoginPlayoneContract.Presenter {
+class LoginPlayonePresenter(
+    val signUpAndSignIn: SignUpAndSignIn,
+    val mapper: UserMapper
+) : LoginPlayoneContract.Presenter {
+
+    var loginView: LoginPlayoneContract.View? = null
```

Comment:
`lateinit var` will be better if we suppose to not set the `null`.
`nullable` is kinda troublesome to handle it in Kotlin, we should use `nonnullable` variable as possible as we can if a variable won't be `null`, I think.

Reply:
I think it can be null....